### PR TITLE
School library app form UI

### PR DESCRIPTION
**Pull request by chepkok3**
- [ ] Created a app.rb file that serves as the console app entry point. It has methods that do the following:
1. List all books.
2. List all people.
3. Create a person (teacher or student, not a plain Person).
4. Create a book.
5. Create a rental.
6. List all rentals for a given person id.
- [ ] In the main.rb I have defined the entry point with a method called main that is invoked at the end of the file. The method does the following:
- Present the user with a list of options to perform.
- Lets users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+require_relative 'app'
```
Comment (hiromataba) [inline main.rb:1]: - [ ] Please fix the error below I am getting 👇  when running the project I am getting it 

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @chepkok3    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (chepkok3) [reply main.rb:1]: @hiromataba Check your machine, please. I have run ruby main.rb and there is no error on my computer. 
Screenshot 2023-04-19 at 21 11 56 . Better yet, you can have someone else run the app.

Comment (hiromataba) [reply main.rb:1]: @chepkok3  can you please try to tell another person to clone your project and check ??
I am tested and cloned a lot of projects to add the basic UI and they are running correctly

**Review by addod19 — APPROVED** [review]
### Hi @chepkok3, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] App runs successfully without any problems :rocket: 
Screenshot from 2023-04-20 08-25-17


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**